tests/generate_packageTest.py

```diff
@@ -0,0 +1,82 @@
+import copy
+import os
+import sys
+import unittest
+from unittest.mock import patch
+from bears.generate_package import (touch, create_file_from_template,
+                                    create_file_structure_for_packages,
+                                    perform_upload, main,
+                                    create_upload_parser)
+
+
+class touchTest(unittest.TestCase):
+
+    def test_file_doesnt_exist(self):
+        touch('TestFile.py')
+        self.assertTrue(os.path.exists('TestFile.py'))
+
+    def test_file_that_exists(self):
```

Comment:
i had to test it if it does exist already, to get full coverage, ideally it should not touch the file.